packages/fast-web-utilities/src/dom.ts

```diff
@@ -1,6 +1,31 @@
 import { canUseDOM } from "exenv-es6";
 import { isBoolean } from "lodash-es";
 
+/**
+ * A test that ensures that all arguments are HTML Elements
+ */
+export function isHTMLElement(...args: HTMLElement[]): boolean {
+    return args.every((arg: HTMLElement) => arg instanceof HTMLElement);
+}
+
+/**
+ * Returns all displayed elements inside of a root node that match a provided selector
+ */
+export function getDisplayedNodes(
```

Comment:
Do you mean rendered instead of displayed?

Reply:
No - this is specifically asserting the display property of HTML nodes. This approach is taken to determine if nodes are displayed because the alternative is to get the computed styles of the element, which forces a layout pass (bad for perf)

https://stackoverflow.com/questions/19669786/check-if-element-is-visible-in-dom

Reply:
Oops, was going to respond with a link here as well: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetParent